blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/22449
+    category: Search
+  - description: "An `sg_service` Postgres role has been introduced, as well as an `sg_repo_access_policy` policy on the `repo` table that restricts access to that role. The role that owns the `repo` table will continue to get unrestricted access."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22303
+    category: Repositories
+  - description: "Every service that connects to the database (i.e. Postgres) now has a \"Database connections\" monitoring section in its Grafana dashboard."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22570
+    category:
+  - description: A new bulk operation to close many changesets at once has been added to Batch Changes.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22547
+    category: Batch Changes
+  - description: Backend Code Insights will aggregate viewable repositories based on the authenticated user.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22471
+    category: Repositories
+  - description: Added support for highlighting .frugal files as Thrift syntax.
+    url:
+    category:
+  - description: "Added `file:contains.content(regexp)` predicate, which filters only to files that contain matches of the given pattern."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22666
+    category:
+  - description: "Repository syncing is now done in streaming mode by default. Customers with many repositories should notice code host updates much faster, with repo-updater consuming less memory. Using the previous batch mode can be done by setting the `ENABLE_STREAMING_REPOS_SYNCER` environment variable to `false` in `repo-updater`. That environment variable will be deleted in the next release."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22756
+    category: Batch Changes
+  - description: Enabled the ability to query Batch Changes changesets, changesets stats, and file diff stats for an individual repository via the Sourcegraph GraphQL API.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22744/
+    category: Batch Changes
+  - description: "Added \"Groovy\" to the initial `lang:` filter suggestions in the search bar."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22755
+    category: Search
+  - description: "The `lang:` filter suggestions now show all supported, matching languages as the user types a language name."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22765
+    category:
+  - description: Code Insights can now be grouped into dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/issues/22215
+    category:
+  - description: "Batch Changes changesets can now be [published from the Sourcegraph UI](https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui)."
```

Comment:
@christinaforney - I don't think we can add links within the changelog. Should we set the docs link as the URL instead?

Reply:
Yep, I think that's a good call.